geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SimpleInstancesExample.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import Canvas from "@metacell/geppetto-meta-ui/3d-canvas/Canvas";
+import CameraControls from "@metacell/geppetto-meta-ui/camera-controls/CameraControls";
+import SimpleInstance from "@metacell/geppetto-meta-core/model/SimpleInstance";
+import { withStyles } from '@material-ui/core';
+import neuron from './SketchVolumeViewer_SAAVR_SAAVR_1_1_0000.obj';
```

Comment:
Obj is missing in repo: Can't resolve './SketchVolumeViewer_SAAVR_SAAVR_1_1_0000.obj'